code/game/machinery/portable_turret.dm

```diff
@@ -902,7 +902,7 @@ Status: []<BR>"},
 
 /obj/machinery/porta_turret_cover/attack_hand(mob/user)
 	. = ..()
-	if(.)
+	if(. || !Parent_Turret)

```

Comment:
nah this is kinda old, a few months. It happened on box some rounds but I'm still not sure why